Readme.md

```diff
@@ -1,155 +1,135 @@
-# To run one specific test:
-$ make check-jit RUNTESTFLAGS="-v -v -v jit.exp=jit.dg/test-asm.cc"
+$ make -C gcc check-jit RUNTESTFLAGS="-v -v -v jit.exp=jit.dg/test-asm.cc"
 ```
 
-**Put the path to your custom build of libgccjit in the file `config.toml`.**
-
-You now need to set the `gcc-path` value in `config.toml` with the result of this command:
-
+Now that you've compiled GCC with libgccjit support, the installed GCC toolchain with `libgccjit.so` can be found in the `gcc-install` directory created above. You will need to provide the path to the specific subdirectory containing `libgccjit.so` to rustc_codegen_gcc. Use the following commands to leave the `gcc-build` directory and find the absolute path within the `gcc-install` directory:
 ```bash
-$ dirname $(readlink -f `find . -name libgccjit.so`)
+$ cd ..
+$ dirname $(readlink -f `find ./gcc-install -name libgccjit.so`)
 ```
 
-and to comment the `download-gccjit` setting:
+If desired, you may now delete the `gcc-source` and `gcc-build` directories to reclaim disk space, but keep the `gcc-install` directory.
 
+Then, within the rustc_codegen_gcc repository directory, create a `config.toml` file with the following contents. Replace `[MY PATH]` with the path you found above.
 ```toml
 gcc-path = "[MY PATH]"
-# download-gccjit = true
 ```
 
-Then you can run commands like this:
+### Building rustc_codegen_gcc
+Now that you have a `config.toml` file set up to use `libgccjit.so`, you can proceed to building the build system and sysroot. `prepare` will retrieve and patch the sysroot source, while `build` will build the sysroot.
 
 ```bash
-$ ./y.sh prepare # download and patch sysroot src and install hyperfine for benchmarking
+$ ./y.sh prepare
 $ ./y.sh build --sysroot --release
 ```
 
-To run the tests:
-
+You may also run the tests:
 ```bash
 $ ./y.sh test --release
 ```
 
 ## Usage
-
-You have to run these commands, in the corresponding order:
-
+The following example commands are run with `$CG_GCCJIT_DIR` representing the path to your rustc_codegen_gcc directory. 
+### Cargo
+To invoke `cargo`, run the following example command:
 ```bash
-$ ./y.sh prepare
-$ ./y.sh build --sysroot
+$ CHANNEL="release" $CG_GCCJIT_DIR/y.sh cargo run
 ```
-To check if all is  working correctly, run:
 
+You may verify your build is working properly by building a test project:
  ```bash
-$ ./y.sh cargo build --manifest-path tests/hello-world/Cargo.toml
-```
-
-### Cargo
-
-```bash
-$ CHANNEL="release" $CG_GCCJIT_DIR/y.sh cargo run
+$ CHANNEL="release" $CG_GCCJIT_DIR/y.sh cargo build --manifest-path $CG_GCCJIT_DIR/tests/hello-world/Cargo.toml
 ```
 
-If you compiled cg_gccjit in debug mode (aka you didn't pass `--release` to `./y.sh test`) you should use `CHANNEL="debug"` instead or omit `CHANNEL="release"` completely.
+Note: If you compiled rustc_codegen_gcc in debug mode (i.e., you didn't pass `--release` to `./y.sh` above), you should use `CHANNEL="debug"` or omit `CHANNEL="release"` completely.
```

Comment:
Would be interesting to instead explain here that the `CHANNEL` environment variable can be used.